src/openrct2-ui/windows/ObjectLoadError.cpp

```diff
@@ -199,7 +201,7 @@ class ObjectDownloader
         }
 
         auto& entry = _entries[_currentDownloadIndex];
-        auto name = String::Trim(std::string(entry.name, sizeof(entry.name)));
+        auto name = String::Trim(std::string(entry.GetName()));
```

Comment:
This assumes that the object download server recognises json-style object names. We need to make sure this is the case before the new save format is merged.

Reply:
Yes, I plan for it to use a different API for JSON object names.